README.md

```diff
@@ -76,6 +76,38 @@ Parasoft Demo Application exposes port 9001 for the user to connect to the HSQLD
 | Username | `SA`                                                |
 | Password | `pass`                                              |
 
+## Using Parasoft JMS/MQ Proxy
+There are two main services for order management in PDA, **order service** and **inventory service**. After an order is submitted, order service would send 
+a request through messaging queue to check and decrease the inventory. After the operation is done, inventory service would send a response through messaging queue which includes the information of the operation result.
+
+To use Parasoft JMS/MQ proxy, you should enable the feature in PDA Demo Administration page and configure the connection queues with your message proxy in Virtualize.
+
+<img src="src/main/resources/static/common/images/services_diagram.png" width="500" alt="services_diagram">
+
+Default connection of **order service**
```

Comment:
Let's get rid of tables from Line 87 to Line 99, and set the default queues into those input fields in demo admin page.

Reply:
And after that, update the diagram to include proxy with the 4 queues to show the workflow.